experimenter/experimenter/templates/changelog/overview.html

```diff
@@ -0,0 +1,17 @@
+{% extends "experimenter_base.html" %}
+
+{% block title %}
+  History
+{% endblock %}
+
+{% block content %}
+    <h1>Changes Overview</h1>
+    <p>Experiment Slug: {{ object.slug }}</p>
+    {% for changelog in object.get_changelogs %}
```

Comment:
You can use `experiment` here instead of `object`

Reply:
The model is named as NimbusExperiment and I'll need to use this in lowercase like this `nimbusexperiment` which doesn't look good or else give it a custom name like I did earlier that's why I just used the generic object here